versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.74"
```

Comment:
Test with lates 4.4.0

Reply:
Done

Reply:
We are using 3.74 in terragrunt, why update to 4.4? @naveen-variant @salyateem13